README.md

```diff
@@ -333,6 +333,38 @@ $results = array(/* ... */);
 $adapter = new FixedAdapter($nbResults, $results);
 ```
 
+### AdaptersAdapter
+
+This adapter can take an array of any other adapters and go through them
```

Comment:
Could you possibly explain further what this adapter does? How does it treat both the number of results and the slice with several adapter?